mdoc-js/src/main/scala/mdoc/modifiers/JsModifier.scala

```diff
@@ -252,7 +252,11 @@ class JsModifier extends mdoc.PreModifier {
 
     runs += code
     new CodeBuilder()
-      .printlnIf(!mods.isInvisible, s"```scala\n${input.text}\n```")
+      .printIf(!mods.isInvisible, s"```scala")
+      .printIf(!remainingMods.isEmpty && !mods.isInvisible, remainingMods.mkString(" ", " ", ""))
+      .printIf(remainingMods.isEmpty && !mods.isInvisible, s"\n")
+      .printIf(!mods.isInvisible, s"${input.text}\n```")
+      .printIf(!mods.isInvisible, s"\n")
```

Comment:
```suggestion
      .printIf(!mods.isInvisible, s"${input.text}\n```")
```
we already have \n, no?

Reply:
If I remove this line, some of the previously existing tests fail, as they assert the `<div--- ` will appear on a line following the code fences. 

Said differently, the previous implementation was a println that printed a line break at the end too - `.printlnIf( stuff here \n ) `... this replicates that behaviour with an extra print("\n") - I could change it back to a println if you feel it's more readable?

Reply:
Ok, let's leave it as is.